procedures/katello/branch.md.erb

```diff
@@ -29,12 +29,7 @@
 ## Release Packager
 
 - [ ] Ensure tool_belt config is merged and output from `./tools.rb koji configs/katello/<%= release %>.yaml` matches expectations
-- [ ] Ensure releases of installer modules, this is typically done as part of Foreman module release workflow
-    - [ ] [candlepin](https://github.com/theforeman/puppet-candlepin)
-    - [ ] [pulp](https://github.com/theforeman/puppet-pulp)
-    - [ ] [pulpcore](https://github.com/theforeman/puppet-pulpcore)
-    - [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
-    - [ ] [katello](https://github.com/theforeman/puppet-katello)
+- [ ] Ensure major changes to Foreman installer modules are completed
```

Comment:
Is it the task of the release packager? I think the release owner is in charge of making sure features are completed. IMHO the goal should be to make sure during stabilization we are really working on making it stable, not finishing features which can destabilize it.

Reply:
@jturel @jlsherrill thought's about this?

Reply:
I think the bullet for the packager would be something like "confirm with release owner that all expected changes are present before performing an installer modules release" - something to ensure that the two are in sync. However it seems rare to me that Katello devs are even slightly aware of the installer changes going in to a release that they didn't ask for themselves so 'expected' is a vague term in my example.

Reply:
That's sort of my point: often a change is made in Katello somewhere and then it turns out to need an installer change. Often that happens during or even after stabilization. My goal is to prevent that. IMHO it's the release manager's job to ensure those changes are in before Foreman stabilizes. The Foreman packager should take care of foreman-installer which means the Katello packager doesn't really have to do anything.

Reply:
I remember one such case not too long ago in fact. 

@jjeffers it sounds like this change becomes:

 - leave the block which you deleted
 - add a new point for the release owner 2 weeks prior to branching to work with the team to ensure all installer changes are done